src/java.desktop/share/classes/javax/swing/plaf/metal/MetalLookAndFeel.java

```diff
@@ -782,6 +807,8 @@ protected void initComponentDefaults(UIDefaults table) {
                           "SPACE", "pressed",
                  "released SPACE", "released"
               }),
+            // margin is (2, 14, 2, 14) which is vastly larger in horizontal
```

Comment:
I see below that there is another comment saying margin is 2 all the way around, what is the difference? how are these comments placed to indicate what margins they are referring to?

Reply:
The below comments apply to check box and radio button; this comment applies to regular push button.

Reply:
Yep. The issue is with regular buttons as Alexey mentioned. This caused a long discussion chain described in the linked issues. Nothing to do with CheckBox or RadioButton.